### PR TITLE
[pyUSB] Update and fix on python 3.9

### PR DESCRIPTION
**Pull request by RedGl0w**
This fixes https://github.com/numworks/epsilon/issues/1742

### CODE REVIEW

File: build/device/usb/__init__.py [hunk 1]
```diff
@@ -1,17 +1,32 @@
-# Copyright (C) 2009-2017 Wander Lairson Costa
```
Comment (EmilieNumworks) [inline build/device/usb/__init__.py:1]: We would rather update pyUSB with one of the released version: https://github.com/pyusb/pyusb/tags

**Review by EmilieNumworks — COMMENTED** [review]
Thanks for fixing this! Could you rebase your PR to reduce it to the two last commits? I can do it if you're ok with me pushing on your fork.

**Comment by RedGl0w** [comment]
> Thanks for fixing this! Could you rebase your PR to reduce it to the two last commits? I can do it if you're ok with me pushing on your fork.

You can do it :p
I don't have any computer yet, so do it if you want to

File: build/device/usb/__init__.py (hunk 1 above)
Comment (RedGl0w) [reply build/device/usb/__init__.py:1]: Indeed, good idea

**Pull request merged**